great_expectations/cli/toolkit.py

```diff
@@ -22,22 +24,31 @@
 from great_expectations.datasource import Datasource
 from great_expectations.exceptions import CheckpointError, CheckpointNotFoundError
 from great_expectations.profile import BasicSuiteBuilderProfiler
+from great_expectations.data_context.types.resource_identifiers import (
+    ExpectationSuiteIdentifier,
+    ValidationResultIdentifier,
+)
 
-
-def create_expectation_suite(
-    context,
-    datasource_name=None,
-    batch_kwargs_generator_name=None,
-    generator_asset=None,
-    batch_kwargs=None,
-    expectation_suite_name=None,
-    additional_batch_kwargs=None,
-    empty_suite=False,
-    show_intro_message=False,
-    open_docs=False,
-    profiler_configuration="demo",
-    data_asset_name=None,
-):
+class MyYAML(YAML):
+    # copied from https://yaml.readthedocs.io/en/latest/example.html#output-of-dump-as-a-string
+    def dump(self, data, stream=None, **kw):
+        inefficient = False
+        if stream is None:
+            inefficient = True
+            stream = StringIO()
+        YAML.dump(self, data, stream, **kw)
+        if inefficient:
+            return stream.getvalue()
+
+yaml = MyYAML()   # or typ='safe'/'unsafe' etc
+
+yaml.indent(mapping=2, sequence=4, offset=2)
+yaml.default_flow_style = False
+
+def create_expectation_suite(context, datasource_name=None, batch_kwargs_generator_name=None, generator_asset=None,
+                             batch_kwargs=None, expectation_suite_name=None, additional_batch_kwargs=None,
+                             empty_suite=False, show_intro_message=False, flag_build_docs=True, open_docs=False,
+                             profiler_configuration="demo", data_asset_name=None):
```

Comment:
I'd like to quickly pair on some changes for readability here, but I think it's not substantively different.

Reply:
Added a WARNING in the doc string based on our pairing session.